content/dart/concepts/strings/strings.md

```diff
@@ -0,0 +1,101 @@
+---
+Title: 'Strings'
+Description: 'A "String" is the text enclosed in quotes, such as "Hello, Dart". It represents words or sentences and is used for handling and manipulating text in coding.'
+Subjects: 
+  - 'Developer Tools'
+  - 'Code Foundations'
+  - 'Computer Science'
+Tags:
+  - 'Methods'
+  - 'Functions'
+  - 'Strings'
+CatalogContent: 
+  - 'learn-dart'
+  - 'paths/computer-science'
+---
+
+A **String** is a sequence of characters \(such as letters, numbers, and symbols) that is used to represent text. It is a data type designed specifically for handling textual data. **Strings** are enclosed in single \(') or double \(") quotes.
+
+Single-line strings are represented using single or double quotes.
+Triple quotes are used to represent multi-line strings.
+
+## Syntax
+
+```
+void main() {
+  String variable_name = 'This is a Dart string with single quotes.';
+  print(variable_name);
+}
+```
+
+```
+void main() {
+  String variable_name = "This is a Dart string with double quotes.";
+  print(variable_name);
+}
+```
+
+```
+void main() {
+  String multiLineString = '''
+    This is a multi-line
+    string in Dart using single quotes.
+    It allows you to write
+    text across multiple lines.
+  ''';
+  print(multiLineString);
+}
+```
+
+```
+void main() {
+  String multiLineString = """
+    This is a multi-line
+    string in Dart using double quotes.
+    It allows you to write
+    text across multiple lines.
+  """;
+  print(multiLineString);
+}
+```
+
```

Comment:
`Shell` output here for the above example using,
\```shell
\`Your code-byte example output`
\```

Reply:
Updated based on comment to include output in specified example @Vino-Siva